tests/js/spec/views/organizationEvents/index.spec.jsx

```diff
@@ -6,13 +6,14 @@ import {mount} from 'enzyme';
 import {setActiveOrganization} from 'app/actionCreators/organizations';
 import GlobalSelectionStore from 'app/stores/globalSelectionStore';
 import OrganizationEventsContainer from 'app/views/organizationEvents';
+import ProjectsStore from 'app/stores/projectsStore';
 
 describe('OrganizationEvents', function() {
   let wrapper;
   const environments = ['production', 'staging'];
   const {organization, router, routerContext} = initializeOrg({
     projects: [
-      {isMember: true, environments},
+      {isMember: true, environments, isBookmarked: true},
```

Comment:
Do tests fail without this case? We should support this without having to specify bookmarked status (assumed to be false anyway)

Reply:
it changes the order 😎

Reply:
tldr; `withProjects` orders items slightly differently for some reason (probably specifically items without names or slugs) and adding a `bookmarked` flag was just an easy way to change the order. I can also try adding slugs to all of the mock projects and see if that moves the needle.